src/less/components/search/index.less

```diff
@@ -1,4 +1,117 @@
 @import (reference) 'mixins';
+@import (reference) './SearchControl';
+
+// ----- LAYOUT -----
+/*
+    auto    1fr    auto
+  +-------+-------+-------+
+  | lbl   | lbl   | lbl   | auto
+  +-------+-------+-------+
+  | icon  | input | btn   | auto
+  +-------+-------+-------+
+  | extra | extra | extra | auto
+  +-------+-------+-------+
+*/
+
+
+hx-search-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: 2rem 1fr auto;
+  -ms-grid-rows: auto auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    'lbl   lbl   lbl'
+    '.     input btn'
+    'extra extra extra';
+  grid-template-columns: 2rem 1fr auto;
+  grid-template-rows: auto auto auto;
+
+  > input[type="search"] {
+    -moz-appearance: textfield;
+    -webkit-appearance: textfield;
+    -ms-grid-column: 2;
+    -ms-grid-row: 2;
+    background: none;
+    border: none;
+    font-size: 1rem;
+    grid-area: input;
+    height: 2rem;
+    min-width: 8rem; // ~16 chars
+    z-index: 1;
+
+    // reset for IE as well
+    &::-ms-clear {
+      display: none;
+      height: 0;
+      width : 0;
+    }
+
+
+    &::-webkit-search-cancel-button {
```

Comment:
Why do we need this?
Which browser is it targeting? (Chrome? Safari? Edge?)

Reply:
Added comment:

```less
// Removes default Safari cancel button
&::-webkit-search-cancel-button {
  display: none;
}
```

Removes this:

![image](https://user-images.githubusercontent.com/10120600/66070120-5505de80-e516-11e9-894b-9ff1e06ff344.png)

Reply:
Interesting. So `-webkit-appearance: textfield;` doesn't change this behavior in Safari?

Reply:
No, applying `-webkit-appearance: textfield;` didn't change the behavior.